graphql-ballerina/tests/06_errors_test.bal

```diff
@@ -52,7 +56,10 @@ isolated function testMissingArgumentValueQuery() returns error? {
 isolated function testEmptyQuery() returns error? {
     string graphqlUrl = "http://localhost:9095/graphql";
     string document = "{ }";
-    json result = check getJsonPayloadFromService(graphqlUrl, document);
+    http:Client httpClient = check new(graphqlUrl);
+    http:Response response = check httpClient->post("/", { query: document });
+    test:assertEquals(response.statusCode, 400);
```

Comment:
Since these lines are duplicated in a few places, we can move this to a separate utility function as well.